scilpy/image/volume_operations.py

```diff
@@ -529,3 +530,69 @@ def crop_data_with_default_cube(data):
     roi_mask = _mask_from_roi(shape, roi_center, roi_radii)
 
     return data * roi_mask
+
+
+def normalize_metric(metric, reverse=False):
+    """
+    Normalize a metric array to a range between 0 and 1,
```

Comment:
This could actually work with an array of any dimension. It does not have to be a volume. Ex, it is probably useful everytime we want to change the range of a colormap.  Should we put it in utils? @arnaudbore